### PR TITLE
Add Teams endpoint

### PR DESCRIPTION
**Pull request by CarlosRodrigo**

### CODE REVIEW

**Review by jvpessoa10 — COMMENTED** [review]
Please, checkout to dev first.

When deleting a team, this routine should delete team's meetings too?

File: src/controllers/teamController.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  validateTeamId (team) {
+    Joi.validate(team, Joi.object({
+      id: Joi.string().required()
+    }))
+  }
+
+  validateFullTeam (team) {
+    Joi.validate(team, Joi.object({
+      id: Joi.string().required(),
+      topics: Joi.array().items(Joi.string()).required(),
```
Comment (viniciusalbuquerque) [inline src/controllers/teamController.js:42]: I think you didn't mean `topics` here. Maybe you mean `members`. What about the `admin`?

File: src/controllers/teamController.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      timestamp: Joi.date().timestamp().required()
+    }))
+  }
+
+  validateTeam (team) {
+    Joi.validate(team, Joi.object({
+      topics: Joi.array().items(Joi.string()).required()
```
Comment (viniciusalbuquerque) [inline src/controllers/teamController.js:49]: Same as above comment.

**Review by viniciusalbuquerque — CHANGES_REQUESTED** [review]
A minor concern

**Pull request closed without merging**